### PR TITLE
SIMSBIOHUB-346: Survey Interfaces - Implement Sticky Header and Breadcrumbs

### PR DESCRIPTION
**Pull request by KjartanE**
## Links to Jira Tickets

- https://apps.nrs.gov.bc.ca/int/jira/browse/SIMSBIOHUB-346

## Description of Changes

- Edit and Create Survey Pages, now have sticky header with breadcrumb trail and save/cancel buttons.

## Testing Notes

- {List any relevant testing considerations, necessary pre-reqs, and areas of the app to focus on. Specifically, include anything that will help the reviewers of this PR verify the code is functioning as expected.}

### CODE REVIEW

**Review by curtisupshall — COMMENTED** [review]
Code looks good. I added a **Do Not Merge** flag to this PR, since you need to change your base branch from `dev-staging` to `dev` (and resolve the subsequent merge conflicts. I think for 99% of them, you can just choose _Accept Incoming Change_)

File: app/src/features/surveys/view/components/SurveyBaseHeader.tsx [hunk 1]
```diff
@@ -0,0 +1,73 @@
...
+  subTitle?: JSX.Element;
+  breadCrumb?: JSX.Element;
+  buttonJSX?: JSX.Element;
+}
+/**
+ * Generic Survey header for all survey views
+ *
+ * @return {*}
+ */
+const SurveyBaseHeader: React.FC<ISurveyHeader> = (props) => {
```
Comment (curtisupshall) [inline app/src/features/surveys/view/components/SurveyBaseHeader.tsx]: Nit: prefer `(props: ISurveyHeader)` instead of `React.FC`

**Review by al-rosenthal — DISMISSED** [review]
ran this locally and it looks good

File: app/src/components/map/components/MapBaseCss.tsx
Comment (NickPhura) [inline app/src/components/map/components/MapBaseCss.tsx:1]: This seems to be a duplicate of `components/map/styles/MapBaseCss.tsx`

File: app/src/features/surveys/view/components/SurveyBaseHeader.tsx [hunk 2]
```diff
@@ -0,0 +1,73 @@
...
+/**
+ * Generic Survey header for all survey views
+ *
+ * @return {*}
+ */
+const SurveyBaseHeader = (props: ISurveyHeader) => {
+  const { title, subTitle, breadCrumb, buttonJSX } = props;
+
+  return (
+    <>
```
Comment (NickPhura) [inline app/src/features/surveys/view/components/SurveyBaseHeader.tsx]: This fragment isnt needed I believe.

**Review by NickPhura — APPROVED** [review]

**Review by al-rosenthal — APPROVED** [review]
🥂

**Pull request merged**